plenum/server/replica.py

```diff
@@ -1631,25 +1631,23 @@ def send(self, msg, stat=None) -> None:
 
     def caught_up_till_pp_seq_no(self, last_caught_up_pp_seq_no):
         self.addToOrdered(self.viewNo, last_caught_up_pp_seq_no)
-        # self._remove_till_caught_up_pp_seq_no(last_caught_up_pp_seq_no)
+        self._remove_till_caught_up_pp_seq_no(last_caught_up_pp_seq_no)
 
     def _remove_till_caught_up_pp_seq_no(self, last_caught_up_pp_seq_no):
```

Comment:
I think it's required. The proof is INDY-70 and the tests in this PR: if we already had enough data for Ordering, but were interrupted by catch-up.
Also I think it may be needed for INDY-13 (catch-up during view-change),